def.bzl

```diff
@@ -149,4 +151,5 @@ def gazelle(name, **kwargs):
         name = name,
         srcs = [runner_name],
         tags = tags,
+        visibility = visibility,
```

Comment:
TBH, if you wanted to just make this single file change and not add any of the other testing I'd be happy to merge it.